lib/fluent/plugin_helper/http_server.rb

```diff
@@ -64,6 +114,16 @@ def stop
 
       private
 
+      def _http_server_overwrite_config(config, opts)
+        conf = config
```

Comment:
Is this for short-cut or forget to call `.dup`?

Reply:
forgot to call dup..
 https://github.com/fluent/fluentd/pull/2787/commits/abb50a853bc309dfa40f546c0f5416e72adb2e8a

Reply:
@repeatedly https://github.com/fluent/fluentd/pull/298/files#diff-d34ab903425415e82b8cd4d891adadddR41
I've found all tests failed in travis because they expected Section class not to have #dup. 
Do you know why it does?

Reply:
Hmm... I don't remember. If all tests are succeded after removed it, we can remove it.